### PR TITLE
updating minecraft helm chart

### PR DESCRIPTION
**Pull request by KeenWill**

### CODE REVIEW

File: labscraft/values.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+labscraft:
+  secret: labscraft
+  eula: "TRUE"
+  version: "1.15.2"
+  type: "SPIGOT"
+  spigotDownloadUrl: https://cdn.getbukkit.org/spigot/spigot-1.15.2.jar
+  difficulty: normal
+  icon: https://s3.amazonaws.com/labscraft.pennlabs.org/logo.png
+  maxPlayers: 30
+  allowNether: false
```
[the remark this answers is not in the document]
Comment (KeenWill) [reply labscraft/values.yaml]: Why? It's not the default value

File: labscraft/values.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  motd: "Welcome to Labscraft!"
+  worldSaveName: labscraft
```
Comment (KeenWill) [inline labscraft/values.yaml]:
```suggestion
  worldSaveName: labscraft-v1
```

File: labscraft/values.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  downloadModpackUrl:  # TODO
```
Comment (KeenWill) [inline labscraft/values.yaml]:
```suggestion
  # downloadModpackUrl:  # TODO
```

File: labscraft/values.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ArmaanT) [reply labscraft/values.yaml]: Because we want to go to the nether?

[the remark this answers is not in the document]
Comment (KeenWill) [reply labscraft/values.yaml]: oh i thought your comment referenced a different line my b

Comment (KeenWill) [inline labscraft/values.yaml]:
```suggestion
```

File: labscraft/values.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  removeOldMods: true
+  memory: 2048M
+
```
Comment (KeenWill) [inline labscraft/values.yaml]:
```suggestion
pvp: true

```

**Comment by KeenWill** [comment]
Ok I think i did everything you asked @ArmaanT

File: labscraft/init.sh [hunk 5]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+helm upgrade --install --atomic --set minecraftServer.eula=true -f values.yaml labscraft pennlabs/labscraft
```
Comment (ArmaanT) [inline labscraft/init.sh]: pennlabs/minecraft not labscraft for the helm chart name

Comment (KeenWill) [inline labscraft/init.sh]:
```suggestion
helm upgrade --install --atomic --set minecraftServer.eula=true -f values.yaml labscraft pennlabs/minecraft
```

**Pull request merged**